Rebus.ServiceProvider/Config/ServiceProviderExtensions.cs

```diff
@@ -23,10 +24,12 @@ public static IServiceProvider StartRebus(this IServiceProvider serviceProvider)
         if (serviceProvider == null) throw new ArgumentNullException(nameof(serviceProvider));
 
         async Task StartHostedServices()
-        {
-            var disposalHelper = serviceProvider.GetRequiredService<RebusDisposalHelper>();
-            var services = serviceProvider.GetServices<IHostedService>().ToList();
+        { 
+            Assembly assembly = Assembly.GetExecutingAssembly();
```

Comment:
What does this line do? 🤔

Reply:
Sorry, this is the test code that I missed.
I have removed it.